docker/build_debian_packages.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir -p ../debs
+
+docker build --tag dh-venv-builder:bionic --build-arg distro=ubuntu:bionic -f Dockerfile-dhvirtualenv .
+#docker build --tag dh-venv-builder:stretch --build-arg distro=debian:stretch -f Dockerfile-dhvirtualenv .
+
+docker run -it --rm --volume=$(pwd)/../\:/synapse/build dh-venv-builder:bionic
+#ocker run -it --rm --volume=$(pwd)/../\:/build dh-venv-builder:stretch
```

Comment:
I think if we're going to keep the alternatives around, we should maintain them:

```suggestion
#docker run -it --rm --volume=$(pwd)/../\:/synapse/build dh-venv-builder:stretch
```

is it right that it